azure-pipelines.yaml

```diff
@@ -27,6 +27,18 @@ pool:
     - ImageOverride -equals MMSUbuntu20.04TLS
 
 steps:
+  - task: UseDotNet@2
+    displayName: 'Install .NET Core SDK'
+    inputs:
+      version: 3.1.x
+      packageType: sdk
+  
+  - task: UseDotNet@2
+    displayName: 'Install .NET 6.0'
+    inputs:
+      version: 6.x
+      packageType: sdk
```

Comment:
Is .NET 6.0 not installed by default?

Reply:
It might be installed by default. If we add a task for installing any dotnet sdk, then all subsequent tasks only use the sdk which we explicitly added and not the default ones.

Reply:
Ref - https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/use-dotnet-v2?view=azure-pipelines#remarks